tests/cypress/e2e/pickers/richtext.cy.ts

```diff
@@ -37,6 +37,8 @@ describe('Picker tests - richtext', () => {
         richText.type('Hello');
 
         const linkModal = richText.openLinkModal();
+        // Test to make sure parsing of existing URLs do not break picker
+        linkModal.getURLField().type('https://www.interregemr.eu/projets/marhetak-fr');
```

Comment:
It used to be the parsing of this specific url leads to the string `/projets/markehtak-pr`. Because of the starting slash, it is then treated as a jcr path where query throws an error and causes issues. Test to make sure this is not happening anymore.